mne/tests/test_import_nesting.py

```diff
@@ -22,7 +22,7 @@
                        and sys.modules[x] is not None)
 bad = scipy_submodules - ok_scipy_submodules
 if len(bad) > 0:
-    out.append('scipy submodules: %s' % list(bad))
+    out |= {'scipy submodules: %s' % list(bad)}
```

Comment:
@AdoNunes @larsoner is this change intended?

Reply:
Yes, this PR showed it was wrong/out of date and this fixes it